app/forms/steps/client/residence_type_form.rb

```diff
@@ -0,0 +1,53 @@
+module Steps
+  module Client
+    class ResidenceTypeForm < Steps::BaseFormObject
+      include Steps::HasOneAssociation
+      has_one_association :applicant
+
+      attribute :residence_type, :value_object, source: ResidenceType
+      attribute :relationship_to_someone_else, :string
```

Comment:
Not sure that this attr is clear enough that we're talking about resident owner, if this is stored directly on applicant.

Maybe: `relationship_to_owner_of_home_address`?
`relationship_to_owner_of_usual_home_address`
`relationship_to_non_partner_owner_of_usual_home_address` to be extremely verbose

Reply:
Sorry I missed your schema PR which is when it would have been actually helpful to raise an issue

Reply:
Good point! I think `relationship_to_owner_of_usual_home_address` would be good enough